### PR TITLE
Admin: Add logo to manufacturers

### PR DESCRIPTION
**Pull request by maximlion**
Added logo to manufacturer model, made a new form `ShuupAdminNoTranslation`, which is to be used by admin forms that do not need/use translation

Refs https://github.com/shuup/shuup/issues/1455

### CODE REVIEW

**Comment by tulimaki** [comment]
Changelog update would have been cool. But will add it as part of the release. 👍

**Comment by maximlion** [comment]
Yeah didn't want to change it because the manufacturer logo is not used anywhere yet.

File: shuup/admin/forms/_base.py [hunk 1]
```diff
@@ -20,3 +21,12 @@ def __init__(self, **kwargs):
             if issubclass(self.fields[field].widget.__class__, AdminFileWidget):
                 self.fields[field].widget = FileDnDUploaderWidget(
                     upload_path="/default", kind="images", clearable=True)
+
+
+class ShuupAdminFormNoTranslation(ModelForm):
```
Comment (chessbr) [inline shuup/admin/forms/_base.py]: DRY. This code is the same as the class above, you should create some mixin for this

File: shuup/admin/modules/manufacturers/__init__.py [hunk 2]
```diff
@@ -41,7 +42,7 @@ def get_menu_entries(self, request):
         ]
 
     def get_required_permissions(self):
-        return get_default_model_permissions(Manufacturer)
+        return get_default_model_permissions(Manufacturer) | get_default_model_permissions(File)
```
Comment (chessbr) [inline shuup/admin/modules/manufacturers/__init__.py]: I don't think this is really needed

Comment (maximlion) [reply shuup/admin/modules/manufacturers/__init__.py]: I'll change it in the next PR.

**Comment by maximlion** [comment]
@chessbr  I'll change it in the next PR.

**Pull request merged**